plugin/pkg/auth/authorizer/node/metrics.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+// NodeAuthorizerSubsystem is a subsystem used by metrics here.
+const NodeAuthorizerSubsystem = "node_authorizer"
+
+var (
+	// NodeAuthorizerActionsDuration is a histogram of duration of graph actions in node authorizer.
+	NodeAuthorizerActionsDuration = metrics.NewHistogramVec(
```

Comment:
is there a reason to export this? do we want things from other packages referencing this?

Reply:
Done